### DISCUSSION TITLE
Fix for restarting after desitionation is full

### DISCUSSION THREAD

**Discussion by benab** [swar/Swar-Chia-Plot-Manager#497] [Q&A] [answered]
My destination drive filled up. I've removed the *.temp files from the temp drive, but Swar won't start again because the max (8 jobs) jobs are still listed as running (at 100%). How can I clear those listed jobs so that it starts more plots?
Thanks in advance!

![image](https://user-images.githubusercontent.com/16981493/118537629-2fddd800-b71b-11eb-9e29-d8b440a9592b.png)

**Comment by farkeytron** [comment, marked as answer]
If you're trying to save the plots... they're lost.
If you're trying to start again, have you killed the PIDs for each chia.exe process in Task Manager? Look at the "pid" column above.

Either that, or reboot your plotting machine.

**Reply by benab** [reply]
First, thanks for taking the time to respond. 
Yes, I realize the plots are lost since I removed the temp files. I realized afterwards that I could have saved them by copying them to another HDD and renaming them. Ya live, ya learn.

Awesome. Killing the PIDs did it. Didn't know it was that simple. Hopefully this helps someone else too.

Is this documented on Swar's page and I didn't see it or are people just supposed to know this?
